server/enforcer.go

```diff
@@ -238,3 +240,18 @@ func (s *Server) RemoveFilteredNamedGroupingPolicy(ctx context.Context, in *pb.F
 
 	return &pb.BoolReply{Res: e.RemoveFilteredNamedGroupingPolicy(in.PType, int(in.FieldIndex), in.FieldValues...)}, nil
 }
+
+func toSlice(arr interface{}) []interface{} {
+	v := reflect.ValueOf(arr)
```

Comment:
I think there are other ways to determine the type. Do not use reflection because its performance is poor. See:

https://github.com/casbin/casbin/blob/1b1168f22dba11e2a5ac8f68792c1a91c362ff41/enforcer.go#L68-L84